docs/STANDALONE_PLUGIN.md

```diff
@@ -0,0 +1,30 @@
+# Standalone Plugin
```

Comment:
I think we could go with `Standalone Mode` instead of `Standalone Plugin`, as every plugin can be ran in standalone, and `Standalone Plugin` kinda indicates that it's some special kind of plugin.